src/main/java/edu/ie3/datamodel/io/factory/input/participant/ChpInputEntityData.java

```diff
@@ -15,10 +15,21 @@
 import java.util.Map;
 import java.util.Objects;
 
+/** The type Chp input entity data. */
```

Comment:
Suggestion: `Data used by {@link ChpInputFactory} used to create instances of {@link ChpInput}.`